### PR TITLE
bsunanda:Phase2-hgx60 Correct BH geometry...

### PR DESCRIPTION
**Pull request by bsunanda**
Also have compatible units in printouts of different DDD objects

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
@bsunanda , would it be possible to substitute  in SimG4Core/Geometry/src/DDG4SolidConverter.cc parameter "s" by something, for example "ss". This is useful to avoid namespace problems with CLHEP.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@civanch I changed "s" to "solid" and also tried to use namespace wherever needed

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@ianna @civanch @dmitrijus Could you approve this PR at your earliest convenience?

File: SimG4Core/Geometry/src/DDG4SolidConverter.cc [hunk 1]
```diff
@@ -39,202 +37,194 @@ DDG4SolidConverter::DDG4SolidConverter()
...
+  std::vector<double> r;
+  std::vector<double> z;
+  std::vector<double>::const_iterator i = (*par_).begin()+2;
   int count=0;
   for(; i!=(*par_).end(); ++i) {
-    LogDebug("SimG4CoreGeometry") << "z=" << *i ;
+    LogDebug("SimG4CoreGeometry") << "z=" << *i/CLHEP::cm;
     z.push_back(*i); ++i;
-    LogDebug("SimG4CoreGeometry") << " r=" << *i ;
+    LogDebug("SimG4CoreGeometry") << " r=" << *i/cm;
```
Comment (ianna) [inline SimG4Core/Geometry/src/DDG4SolidConverter.cc]: @bsunanda - please use namespase

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@ianna @civanch @dmitrijus I have updated PR by adding the namespace in one statement as suggested by Ianna. Could you approve this PR at your earliest convenience?

**Comment by ianna** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@dmitrijus Please approve this PR

**Pull request merged**